File: DSA/Krish_Mishra_590015053/Day16_Question3.py
def find_maximum(array):
    max_value = array[0]
    for num in array:
        if num > max_value:
            max_value = num
    return max_value
user_input = input("Enter the elements of the array separated by spaces: ")
array = list(map(int, user_input.split()))
if len(array) == 0:
    print("Array is empty. Cannot determine maximum value.")
else:
    max_element = find_maximum(array)
    print(f"The maximum element in the array is: {max_element}")

File: DSA/GEHNA_KHANNA_590013140/day16question3.c
#include <stdio.h>

int findMaximum(int arr[], int size) {
    int max = arr[0];  
    for (int i = 1; i < size; i++) {
        if (arr[i] > max) {
            max = arr[i];  
        }
    }
    return max;
}

int main() {
    int n;

    printf("Enter the number of elements in the array: ");
    scanf("%d", &n);

    int arr[n];  


    printf("Enter %d elements:\n", n);
    for (int i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }

    int maxElement = findMaximum(arr, n);
    printf("The maximum element in the array is: %d\n", maxElement);

    return 0;
}
